### ISSUE TITLE
Error: PR does not contain metadata, nothing to do.

### ISSUE THREAD

**Issue by Dreamsorcerer** [dependabot/fetch-metadata#71]
On this PR: https://github.com/aio-libs/aiohttp-demos/pull/141

I'm getting `Error: PR does not contain metadata, nothing to do.`

What am I doing wrong?

**Comment by somewhatabstract** [comment]
It would be great if this action supported the standard dependabot configuration options, allowing for prefixes to be applied.

**Comment by somewhatabstract** [comment]
This line is the issue:

https://github.com/dependabot/fetch-metadata/blob/27341dbf0ea39493f6b2be1a0c6f4f3270622003/src/dependabot/update_metadata.ts#L10

It needs updating to support a prefix text.

**Comment by mwaddell** [comment]
I agree with @somewhatabstract - this is caused by https://github.com/dependabot/dependabot-core/issues/4652 and cannot be fixed in `fetch-metadata`

**Comment by brrygrdn** [comment]
@Dreamsorcerer Unfortunately this isn't something we can fix in this project, @mwaddell is correct that we have a downstream issue in core to look at how we handle refreshing commit messages.
